OasisDEX/oasis-borrow
# Aave DPM positions on owner page

# [Aave DPM positions on owner page](https://app.shortcut.com/oazo-apps/story/6939/fetch-data-for-aave-multiply-position-in-my-position-view)

## Not Included

- mapping for position type, every new dpm positon will be displayed as multiply
- automation handling / displaying

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- created `aaveDpmPositions$` observable which returns information about dpm positions on aave
- moved few observables from `AaveContext` to `AppContext` as they were required there
  
## How to test 🧪
  <Please explain how to test your changes>

- currently there is no way of testing it due to the lack of aave multiply open flow using dpm, if available create position and check whether is displayed in multiply section on owner page as well as in pie chart
